AdGoBye/Indexer.cs

```diff
@@ -290,6 +290,7 @@ public static void RemoveFromIndex(string path)
     public static void PatchContent(Content content)
     {
         if (content.Type is not ContentType.World) return;
+        Logger.Information("Processing {ID} ({director})", content.Id, content.VersionMeta.Path);
```

Comment:
```suggestion
        Logger.Information("Processing {ID} ({directory})", content.Id, content.VersionMeta.Path);
```
Looks like you dropped a letter